tools/kubernetes/helm/hue/templates/prometheus-alert-rule.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.alertrule.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: example
+    role: alert-rules
+  name: prometheus-example-rules
+spec:
+  groups:
+    - name: hue-pod-monitor.alert.rules
+      rules:
+      - alert: hue_pods_are_down
+        expr: sum(up{job="hue",namespace="data-platform-staging"}) < 2 #Based on the replicas mentioned in values.yaml
+        for: 30m #Change this based on your SLA
+        labels:
+        #Labels required for your alert recognition 
+          severity: High
+        annotations:
+          summary: Hue pods are down 
```

Comment:
Renaming? "Some Hue pods are down"

(as we currently only check that all the pods are down)